version/version.go

```diff
@@ -51,10 +51,28 @@ type ServerInfo struct {
 // MeshInfo contains the versions for all Istio control plane components
 type MeshInfo []ServerInfo
 
+// NodeType decides the responsibility of the proxy serves in the mesh
+type NodeType string
+
+const (
+	// SidecarProxy type is used for sidecar proxies in the application containers
+	SidecarProxy NodeType = "sidecar"
+
+	// Router type is used for standalone proxies acting as L7/L4 routers
+	Router NodeType = "router"
```

Comment:
can we call it gateway? users do not really know what router is, its more of a legacy term for internals.

Reply:
Thanks, revised.